testing/test_submit_scripts/test_submit_emod3d.py

```diff
@@ -30,38 +28,34 @@ def test_main(set_up, mocker):
     mocker.patch(
         "scripts.set_runparams.utils.load_yaml",
         lambda x: mocked_load_yaml(
-            os.path.join(
-                os.path.dirname(os.path.realpath(__file__)),
-                "..",
-                "..",
-                "templates",
-                "gmsim",
-                "16.1",
-                "emod3d_defaults.yaml",
-            )
+            Path(__file__).resolve().parent
+            / ".."
+            / ".."
+            / "templates"
+            / "gmsim"
+            / "16.1"
+            / "emod3d_defaults.yaml"
         )
-        if "emod3d_defaults.yaml" in x
+        if str(x).find("emod3d_defaults.yaml") != -1
         else mocked_load_yaml(x),
     )
 
     for root_path, realisation in set_up:
-        args = get_input_params(
-            root_path, "{}_{}".format("submit_emod3d.py", function), params
+        rel_dir = (
+            Path(root_path)
+            / f"CSRoot/Runs/{get_fault_from_rel(realisation)}/{realisation}"
```

Comment:
Having a single string with '/' here defeats the purpose of using the Path join syntax.
This applies to test_submit_hf too

Reply:
Agreed. Going back to old faithful os.path and str